tests/integration/test_outcome.py

```diff
@@ -298,6 +302,7 @@ def test_outcome_forwarding(
         "reason": "project_id",
         "event_id": event_id,
         "remote_addr": "127.0.0.1",
+        "data_categories": ["error"],
```

Comment:
If the event_id is a single event, would that event have more than one category?

Reply:
Good catch! There could be multiple outcomes, in fact, but for each outcome there should only ever be a single category. See one of the previous comments for an example.